lib/src/main/java/graphql/nadel/engine/transform/partition/NadelFieldPartitionException.kt

```diff
@@ -0,0 +1,16 @@
+package graphql.nadel.engine.transform.partition
+
+import graphql.ErrorClassification
+import graphql.nadel.error.NadelGraphQLErrorException
+
+class NadelPartitionGraphQLErrorException(
+    message: String,
+    path: List<Any>? = null,
+    errorClassification: ErrorClassification? = partitioningErrorClassification,
```

Comment:
Don't think `errorClassification` is ever variable, so we should hard code it into the super call i.e.

```
class NadelPartitionGraphQLErrorException(
    message: String,
    path: List<Any>? = null,
) : NadelGraphQLErrorException(message, path, partitioningErrorClassification)
```